tests/admin/test-class-admin-apple-json.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+use Apple_Exporter\Settings as Settings;
```

Comment:
Global: `use X\Y\Z as Z` can just be expressed as `use X\Y\Z`. The `as Z` is assumed if you aren't aliasing it to a different name.